sysbrokers/IB/config/ib_config_futures.csv

```diff
@@ -265,7 +270,7 @@ HSCEI-DIV,HHIDPI,HKFE,HKD,50,1,FALSE
 HUF,HUF,CME,USD,30000000,1,FALSE
 HUFEUR,EHF,CME,EUR,30000000,1,FALSE
 IBEX_mini,IBEX,MEFFRV,EUR,1,1,FALSE
-IBXEX,IBEX35,MEFFRV,EUR,10,1,FALSE
+IBEX,IBEX35,MEFFRV,EUR,10,1,FALSE
```

Comment:
Fine with me, but if anyone else has this in their dataset already everything will break.

Reply:
Don't care much about this. We can continue calling it IBXEX. I just figured it was probably a typo.